main.go

```diff
@@ -13,7 +11,6 @@ func main() {
 	rootCmd := cmd.NewRootCmd(version)
 
 	if err := rootCmd.Execute(); err != nil {
-		log.WithError(err).Error(err.Error())
-		os.Exit(1)
```

Comment:
Personally I prefer the explicit exit since a logger is often an interface and has no guarantee that it will exit.
In this case I see no issue.